config/rbac/metrics_auth_role_binding.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: proxy-rolebinding
+  name: metrics-auth-rolebinding
```

Comment:
shouldn't this have also labels?
    app.kubernetes.io/name: build-service
    app.kubernetes.io/managed-by: kustomize